PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,16 @@
-[Explain what your PR does here]
+<!--
+  - Thanks for submitting code to Oppia! Please fill out the following as part of
+  - your pull request so we can review your code more easily.
+  -->
 
-**Checklist**
+## Explanation
+<!--
+  - Explain what your PR does and please include #bugnum in explanation.
```

Comment:
Explain what your PR does. If this PR fixes an existing bug, please include "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue when this PR is merged.

Reply:
I think we are including #bugnum here to make a reference of this PR in the issue comment thread.

Reply:
Done!